.github/workflows/release-cli.yml

```diff
@@ -86,9 +91,140 @@ jobs:
           name: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
           path: bin/*
 
+  package-deb:
+    name: Build deb package
+    needs: build
+    strategy:
+      matrix:
+        include:
+          - GOOS: linux
```

Comment:
I think you don't need to keep `GOARCH` and `pkgArch` separate since they have the same value. Also, let's use `os` and `arch` instead of `GOOS` and `GOARCH` since we're not in a go context anymore.